customer-node/data_access/flightda.js

```diff
@@ -0,0 +1,16 @@
+const knex = require('../knex');
+
+const findFlights = async (source, destination, date) => {
+  try {
+    const flights = await knex('Flight').where({
+      sourceAirport: source,
+      destinationAirport: destination,
+      departureDate: date,
+    });
+    return flights;
+  } catch (err) {
+    return [];
```

Comment:
How to differentiate between an error and just not having any flights with the 3 fields?

Reply:
added 400 errors